packages/preview/light-cv/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "light-cv"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Ansgar Lichter"]
+license = "MIT"
+description = "Minimalistic CV template for your own CV. Please install the font awesome fonts on your system before using the template."
+repository = "https://github.com/AnsgarLichter/cv-typst-template"
+keywords = ["cv", "curriculum vitae", "computer science", "application", "resume"]
+categories = ["cv"]
+exclude = ["example.png"]
```

Comment:
There was a slight misunderstanding I think. The thumbnail `example.png` is automatically excluded. The one I was referring to is `template/media.image.png`, which will be included in the template bundle and cloned into the user's project, even though it is unused. Since it's only cutout of the proper `example.png`, it can even just be deleted rather than excluded, I think.

Other than that, this looks good to merge.

Reply:
It's fixed now.  Thank you!